docs/index.md

```diff
@@ -22,7 +21,7 @@ You should use the Registry if you want to:
 
  * tightly control where your images are being stored
  * fully own your images distribution pipeline
- * integrate images storage and distribution into your inhouse development workflow
+ * integrate images storage and distribution tightly into your inhouse development workflow
```

Comment:
Not related to your change, but:

s/images storage/image storage/
s/inhouse/in-house/